src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -275,6 +278,33 @@ export const BorrowInput = ({
     isEstimatedSweepFeesDataSuccess,
   ])
 
+  const { data: _isSmartContractAddress, isLoading: isAddressByteCodeLoading } = useQuery({
+    queryKey: [
+      'isSmartContractAddress',
+      {
+        userAddress: collateralAccountId
```

Comment:
could memoize the `userAddress` to use throughout this `useQuery` and in `disableSmartContractDeposit`?

Reply:
adffb67841e166bac0eea41ef1fcbb3448136a67